dashboards-reports/public/components/main/report_details/report_details.tsx

```diff
@@ -215,8 +215,8 @@ export function ReportDetails(props: { match?: any; setBreadcrumbs?: any; httpCl
       timeRangeMatcher
     );
 
-    fromDateString = fromDateString.replace(/[']+/g, '');
-    toDateString = toDateString.replace(/[']+/g, '');
+    fromDateString = fromDateString.replace(/[']+/g, '').replace('%2F','/');
+    toDateString = toDateString.replace(/[']+/g, '').replace('%2F','/');
```

Comment:
do we need `decodeURIComponent()` here as well?

Reply:
Argh..I totally forgot about it. As for me it should be similar to `context_menu_helpers.js` as the code looks much the same